CHANGELOG.md

```diff
@@ -4,6 +4,58 @@
 
 <!-- next version -->
 
+## v0.16.0
+
+This release includes version 0.111.0 of the upstream Collector components.
+
+The individual upstream Collector changelogs can be found here:
+
+v0.111.0:
+
+- <https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.111.0>
+- <https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.111.0>
+
+<details>
+<summary>Highlights from the upstream Collector changelog</summary>
+</br>
+
+### 🛑 Breaking changes 🛑
+
+- `service/telemetry`: Change default metrics address to "localhost:8888" instead of ":8888" ([#11251](https://github.com/open-telemetry/opentelemetry-collector/pull/11251))
+  This behavior can be disabled by disabling the feature gate 'telemetry.UseLocalHostAsDefaultMetricsAddress'.
+- `config`: Move component.UseLocalHostAsDefaultHost feature gate to stable. ([#35569](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35569))
```

Comment:
Maybe I would add a note here, that this option cannot be used anymore, since it cannot be disabled

Reply:
i clarified it a bit more, should be fixed now